src/inbox/mod_inbox_muc.erl

```diff
@@ -115,6 +115,7 @@ handle_incoming_message(Host, Room, To, Packet) ->
             mod_inbox_utils:write_to_receiver_inbox(Host, Room, To, Packet)
     end.
 
-%% Returns false, if host is s2s host
+%% Returns true, if host is defined in hosts in the config file.
+%% Returns false, if host is s2s host.
```

Comment:
I think we can remove those comments as well. 
The function is quite clear about what it does